src/nupic/algorithms/SDRClassifier.cpp

```diff
@@ -318,20 +340,18 @@ void SDRClassifier::load(istream &inStream) {
   // Check the version.
   UInt version;
   inStream >> version;
-  NTA_CHECK(version <= 1);
+  NTA_CHECK(version == 2);
```

Comment:
agree, we don't need to be backward compatible with old versions of serialization.